test/test_python_dispatch.py

```diff
@@ -302,6 +302,7 @@ def test_basic(self) -> None:
 $0 = input('x')
 $1 = torch._ops.aten.mul.Tensor($0, $0)
 $2 = input('grad_y')
```

Comment:
backward() now has an extra native op on its call path so needed to add that here